Readme.md

```diff
@@ -0,0 +1,181 @@
+  </ul>
+</details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/Javascript">Javascript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://dart.dev/tools/linter-rules">LINTERS</a></li>
+  </ul>
+</details>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abiy Hussen**
+
+- GitHub: [@githubhandle](https://github.com/abiy006)
+- Twitter: [@twitterhandle](https://twitter.com/Abiy62463489)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abiy-hussen-aman-194587183/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[linter_implementation_with_html_css_javaScript]**
+- **[linter_implementation_with_gitignore]**
+- **[implementing_standard_readme_file]**
+- **[implementing_figma_design_background]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://abiy006.github.io/Capstone-1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+* To clone the repository
+* Double click the index.html file and open in your favourite browser
+
+### Prerequisites
+
+In order to run this project you need:
+
+* To clone the repository
+* A browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone https://github.com/abiy006/Capstone-One.git
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[linter_implementation_with_javascript]**
+- [ ] **[linter_implementation_with_html_css_javascript]**
+- [ ] **[linter_implementation_with_ruby]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Walkthrough
+
+Brife description of this portofolio walkthrough until at this point 
+
+- Walkthrough: [Project Walkthrough](https://www.loom.com/share/fb12380aa2964c4f9641a18f9aea6c19)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to Microverse for giving this inspiring project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
```

Comment:
- Hey, the license link is currently broken, let's add a functional link to improve our readme making it more professional.